modules/olmv1-adding-a-catalog.adoc

```diff
@@ -16,19 +16,22 @@ To add a catalog to a cluster for {olmv1-first} usage, create a `ClusterCatalog`
 .Example `my-redhat-operators.yaml` file
 [source,yaml,subs="attributes+"]
 ----
-apiVersion: catalogd.operatorframework.io/v1alpha1
+apiVersion: olm.operatorframework.io/v1
 kind: ClusterCatalog
 metadata:
-  name: my-redhat-operators
+  name: my-redhat-operators <1>
 spec:
+  priority: 1000 <2>
   source:
-    type: image
     image:
-      ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
-      pollInterval: <poll_interval_duration> <2>
+      pollIntervalMinutes: 10 <3>
```

Comment:
@michaelryanpeter pollIntervalMinutes is different with the previous release.
if we do not want to poll it, do not set it. and there is no default value.

FYI https://github.com/operator-framework/operator-controller/blob/dd4d61fffa399470cf49b5506cf54c0175b40fd3/catalogd/api/v1/clustercatalog_types.go#L345-L351

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/88848/files#diff-37c51032cc762a74feb1f038c6e924252809d045fd71909be26377b6fc82b4edR33